main/FatesIOVariableKindMod.F90

```diff
@@ -52,6 +52,15 @@ module FatesIOVariableKindMod
   character(*), parameter, public :: site_elage_r8 = 'SI_ELEMAGE_R8'
 
 
+  integer, parameter, public :: upfreq_dyn = 1         ! dynamics variables
+  integer, parameter, public :: upfreq_hifr = 2        ! high frequency variables
+  !integer, parameter, public :: upfreq_dyn_multi = 3  ! dynamics multi-dimension
+  integer, parameter, public :: upfreq_hydr = 4        ! Hydro variables
+  integer, parameter, public :: upfreq_nutr = 5
+  integer, parameter, public :: upfreq_hifr_multi = 6   ! high frequency multi-dim
```

Comment:
This appears to be pulled over from #1119.  Since they're not used it doesn't seem to be a big deal, but it might be worth removing for that forthcoming PR,